### DISCUSSION TITLE
Changing the parameters of the css styles

### DISCUSSION THREAD

**Discussion by uPagge** [umputun/remark42#1217] [Ideas]
It would be convenient if it was possible to change the css color values through some external file or docker variable

<img width="1078" alt="Снимок экрана 2022-01-06 в 14 18 20" src="https://user-images.githubusercontent.com/40397740/148375024-296f4203-ceec-4eeb-885e-fe4833be5508.png">

In my example, I need to change the block color to yours in order to fit remark42 into the design. It would be better if there was some kind of file, remark-42.sass, where you could set different colors for buttons, background, etc., it is even possible to redefine icons, and specify it for docker image.

That is, my idea is in simple costomization without the need to assemble my own version of the front

**Comment by Mavrin** [comment]
I can recommend to read this https://github.com/umputun/remark42/issues/5 and this https://github.com/umputun/remark42/pull/599 issues.
